test/lib/commands/publish.js

```diff
@@ -876,49 +876,61 @@ t.test('latest dist tag', (t) => {
     // this needs more than one item in it to cover the sort logic
     { version: '50.0.0' },
     { version: '100.0.0' },
+    { version: '101.0.0', deprecated: 'oops' },
```

Comment:
101.0.0 is used in a test below.
```suggestion
    { version: '102.0.0', deprecated: 'oops' },
```

Reply:
Good catch, I'll amend my commit since this PR has multiples I'll be rebasing.